dev/TabView/TestUI/TabViewPage.xaml

```diff
@@ -142,7 +142,10 @@
                         </StackPanel>
                     </controls:TabViewItem>
 
-                    <controls:TabViewItem x:Name="SecondTab" Header="SecondTab" IsClosable="True">
+                    <controls:TabViewItem x:Name="SecondTab" Header="SecondTab (in green)" IsClosable="True">
+                        <controls:TabViewItem.Resources>
+                            <SolidColorBrush x:Key="TabViewItemHeaderForeground" Color="Green" />
```

Comment:
Perhaps a test to verify this visual behaviour is needed?

Reply:
I looked at PR https://github.com/microsoft/microsoft-ui-xaml/pull/2113 when creating this one and there it apparently sufficed to just set the Foreground color on a specific TopNavViewItem and naming said item accordingly.

However, if a test should be added I woud look into this.

Reply:
Out of context, it just seems confusing to have the second tab as green in the Test UI. Perhaps a test or comment to explain this would be good.

Reply:
I could also remove this "test" if it adds more confusion.

Reply:
I think ideally we would have something in place so that if this regressed we would know about it immediately, That being said I'm not sure it is likely to regress.

Reply:
This addition is causing test failures anyway. I would have to modify at least two tests: 
* InteractionTests.TabViewTests.ToolTipDefaultTest
* InteractionTests.TabViewTests.TabSizeAndScrollButtonsTest

Given this and the comments above I'm starting to think it might be better to just remove this "testing tab" (in other words: revert this SecondTab change).

I'm also not sure right now how an automated test for this resource could look like 🤔

Reply:
> I'm also not sure right now how an automated test for this resource could look like 🤔

Walk visual tree until you find the TextBlock that renders the text and check the foreground of that element. It's not great but it would definitely find out if we have set the color correctly.